README.md

```diff
@@ -0,0 +1,37 @@
+# Link All The Languages!
+-------------------------
+
+A big bundle of different languages all being invoked from a C program.
+
+## Why?
+
+We want to see if we can!
```

Comment:
The idea is that if we can link with C in all of these languages, we can link any of these languages in any order even without C. Not that this statement is inaccurate.

Reply:
Done.